pages/guides/developing/stream-via-browser.en-US.mdx

```diff
@@ -168,8 +168,34 @@ const iceServers = [
   },
 ];
 
+// get user media from the browser (which are camera/audio sources)
+const mediaStream = await navigator.mediaDevices.getUserMedia({
+  video: true,
+  audio: true,
+});
+
 const peerConnection = new RTCPeerConnection({ iceServers });
 
+// set the media stream on the video element
+element.srcObject = mediaStream;
+
+const newVideoTrack = mediaStream?.getVideoTracks?.()?.[0] ?? null;
```

Comment:
This is probably an excessively conservative line of code (under what circumstances would you have a `mediaStream` with a `getVideoTracks` property that's _not_ a function?) but it matches what we have in Livepeer.js